src/region/RegionAnnotations.tsx

```diff
@@ -62,7 +64,9 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
     };
 
     handleStart = (): void => {
-        this.setStaged(null);
+        const { page } = this.props;
+
+        this.setStaged({ location: page, shape: null });
```

Comment:
I don't think the `shape` property is allowed to be null currently. Typescript might be losing sight of that here due to the merging we're doing in `setStaged`.

Reply:
> I don't think the `shape` property is allowed to be null currently. Typescript might be losing sight of that here due to the merging we're doing in `setStaged`.

I actually changed shape to `Rect | null` in CreatorItem type but remain it to `Rect` in CreateArg. Do you think it's ok?